src/components/Dropdown/Dropdown.tsx

```diff
@@ -31,25 +30,26 @@ export const dropdownClassNames = {
 export interface DropdownLabelProps extends HtmlLabelProps {}
 
 export interface DropdownItemProps {
-  /** Operation to run on select */
-  onSelect: () => void;
+  value: string;
   /** Item content */
   children: ReactNode;
+  className?: string;
```

Comment:
Should these `value` and `className` props have also JSDoc?

Reply:
Yes, added.